gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -379,9 +377,15 @@ private static ByteString getFooterContent(
     }
   }
 
+  // Estimates a read time out based on read speeds
+  private static long getReadTimeoutMillis(
```

Comment:
We should add a test for this as this is now dynamic

Reply:
Done